sugar/dsn.go

```diff
@@ -1,32 +1,28 @@
 package sugar
 
+import "net/url"
+
 // Usage of this package
 //
 // db, err := ydb.Open(
 //   ctx,
 //   sugar.DSN("endpoint", "database", false),
 // )
 
-type connectionString struct {
-	endpoint string
-	database string
-	secure   bool
-}
+// DSN makes connection string (data source name) by endpoint, database and secure
+func DSN(endpoint, database string, secure bool) (s string) {
+	qp := url.Values{}
+	qp.Set("database", database)
 
-func (cs connectionString) Build() (s string) {
-	s = "grpc"
-	if cs.secure {
-		s += "s"
+	dsn := url.URL{
+		Scheme:   "grpc",
+		Host:     endpoint,
+		RawQuery: qp.Encode(),
 	}
-	return s + "://" + cs.endpoint + "/?database=" + cs.database
-}
 
-// DSN makes connection string (data source name) by endpoint, database and secure
-func DSN(endpoint, database string, secure bool) (s string) {
-	cs := connectionString{
-		endpoint: endpoint,
-		database: database,
-		secure:   secure,
+	if secure {
+		dsn.Scheme = "grpcs"
 	}
```

Comment:
It makes sense to make this default value true.
```suggestion
	if !secure {
	    dsn.Scheme = "grpc"
	}
```

Reply:
here there are no default value this is required parameter